### PR TITLE
Write JSON stats file in streaming fashion and omit `chunks` for brevity

### PR DESCRIPTION
**Pull request by ghengeveld**
Issue: -

## What I did

Some people have reported out-of-memory errors when using `--webpack-stats-json` on their very large Storybook. This was tracked to `JSON.stringify` using too much memory. This PR addresses that by using [json-ext](https://github.com/discoveryjs/json-ext)'s `stringifyStream` and piping the result into the target file.

I've also omitted `chunks` from the data, because it accounts for about half of the total blob size and isn't particularly useful to have. Technically that's a breaking change but it's very unlikely anybody is actually using that field.

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots? no
- [ ] Does this need a new example in the kitchen sink apps? no
- [ ] Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,6 +120,7 @@
     "@babel/preset-typescript": "^7.12.7",
     "@babel/runtime": "^7.12.5",
     "@compodoc/compodoc": "^1.1.14",
+    "@discoveryjs/json-ext": "^0.5.3",
```
Comment (shilman) [inline package.json:123]: why is this needed?

Comment (ghengeveld) [reply package.json:123]: I don't know, I just thought we were supposed to put it at the root as well.

**Review by tmeasday — APPROVED** [review]
LGTM apart from maybe @shilman's thing.

File: package.json (hunk 1 above)
Comment (shilman) [reply package.json:123]: i don't know what--if anything--needs to be at the root. we should probably clean it out at some point. in the meantime, let's not make it worse unless it's needed.

**Comment by anilanar** [comment]
This works!
